README.md

```diff
@@ -25,9 +25,26 @@ Usage: gitrise [options]
 
 Please see [here](https://api-docs.bitrise.io/#/) for Bitrise API Reference 
 
-## Tests
+## Contributing
+
+Bug reports and suggestions for improvement are always welcome! Pull requests are also accepted!
+
+If you are interested in adding functionality through a pull request, please open a new issue first so that we have the chance to discuss it first.
+
+Before opening a PR, please make sure you have gone through the following steps:
+
+ * have linted the scripts you have touched using [ShellCheck] (https://github.com/koalaman/shellcheck)
+ * have added tests for your changes
```

Comment:
Same as previous: you don't need the word `have`.

Reply:
done!